apps/prairielearn/src/ee/pages/institutionAdminGeneral/institutionAdminGeneral.html.ts

```diff
@@ -15,11 +16,14 @@ type InstitutionStatistics = z.infer<typeof InstitutionStatisticsSchema>;
 
 export function InstitutionAdminGeneral({
   institution,
+  availableTimezones,
   statistics,
   planGrants,
   resLocals,
 }: {
   institution: Institution;
+  authn_providers: (string | null)[];
```

Comment:
AFAICT this is unused on this page?

Reply:
You are correct. This is leftover from when I was originally trying to add/edit authn providers. It has now been removed in commit [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6).